src/tests/indexer/unit/substrateIndexer.test.ts

```diff
@@ -1,6 +1,7 @@
+//@ts-nocheck
```

Comment:
Why do you need this?

Reply:
this is not updated at all at is getting in the way to run the build command.

Reply:
Build command shouldn't even look at the tests folder, but let's leave to another PR